flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -345,7 +337,7 @@ public void done(boolean canceled) {
 		/**
 		 * 1) Create a snapshot object from RocksDB.
 		 *
-		 * @param checkpointId id of the checkpoint for which we take the snapshot
+		 * @param checkpointId        id of the checkpoint for which we take the snapshot
```

Comment:
I think we don't do these kind of alignments because they increase the maintenance overhead when renaming parameters.